### ISSUE TITLE
Running the AuthenticationAuthorizationManager results in an error

### ISSUE THREAD

**Issue by mohamed-elsabagh** [symbiote-h2020/SmartSpaceMiddleware#1]
Hello, I am trying to follow the wiki page in order to add smart space, I reached the step of Setting up the SSP Authentication and Authorization Manager (SspAAM aka LocalAAM)
I managed to creake the saam-keystore.p12, and I added both bootstrap.properties and the .p12 file with the generated jar file but when I start running the jar file I get an error
 
```2018-07-11 15:48:44.706 ERROR [AuthenticationAuthorizationManager,,,] 6319 --- [           main] e.h.s.s.s.h.CertificationAuthorityHelper : java.io.FileNotFoundException: class path resource [home/ubuntu/projects/symbiote/AuthenticationAuthorizationManager/build/libs/saam-keystore.p12] cannot be opened because it does not exist```

I am sure I placed the file in this location and here is this part defined in bootstrap.properties file 

```aam.security.KEY_STORE_FILE_NAME=/home/ubuntu/projects/symbiote/AuthenticationAuthorizationManager/build/libs/saam-keystore.p12```

**Comment by posymeon** [comment]
I'm having the same problem.
I followed the steps from the wiki, tried many different paths and setups to get it to run, the path is recognized, but the keystore .p12 file which resides in the path specified is not found. 

Did you by any chance come with a solution?

The build i'm using is the following:
Java version : "1.8.0_181"
mongoDB: version 3.6
gradle: version 4.6
AuthenticationAuthorizationManager-5.0.3-run.jar
OS: Ubuntu 18.04 server 

The .p12 file has been generated by following the wiki as well.
The bin used for the certificate generation is the following:
"SymbIoTeSecurity-27.6.0-helper.jar"

The error i get is the following:

`2018-08-31 15:05:28.281 ERROR [AuthenticationAuthorizationManager,,,] 9897 --- [           main] e.h.s.s.s.h.CertificationAuthorityHelper : java.io.FileNotFoundException: class path resource [opt/symbiote/SymbioteSmartSpace/AuthenticationAuthorizationManager/build/libs/saam-keystore.p12] cannot be opened because it does not exist`

Part of the bootstrap.properties file:
`# absolute path to the saam-keystore.p12 file
aam.security.KEY_STORE_FILE_NAME=/opt/symbiote/SymbioteSmartSpace/AuthenticationAuthorizationManager/build/libs/saam-keystore.p12
`

**Comment by mohamed-elsabagh** [comment]
yes I just placed the saam-keystore.p12 in the resources folder of the project, the resources folder is defined in class path so it should work

**Comment by posymeon** [comment]
Can you please provide the exact path of the resources folder, as well with the bootstrap.properties settings in order for this to work? 

I placed the saam-keystore.p12 file in the "/AuthenticationAuthorizationManager/build/resources" folder

changed the aam.security.KEY_STORE_FILE_NAME to the absolute path of the file , also tried the relative path, tried many different path configurations...and i always get the same error.

**Comment by posymeon** [comment]
Nevermind, i managed to run the service by placing the keystore file in the src/main/resources folder like you said and by building the AAM again.

I can successfully run the service, but i get the following error when trying 
"http://localhost:8443/get_available_aams"

 "ERROR [AuthenticationAuthorizationManager,38b24b14c51c28b0,38b24b14c51c28b0,true] 7282 --- [nio-8443-exec-6] e.h.s.security.services.AAMServices      : Couldn't establish connection with CoreAAM... falling back to local configuration"

**Comment by mohamed-elsabagh** [comment]
I am sorry I don't know this error, but I think there is a problem of connection between your machine and the back end server, check the connection and check that you are using the right keys

**Comment by mohamed-elsabagh** [comment]
@posymeon  please check a solution to your problem at https://github.com/symbiote-h2020/SymbioteSmartSpace/issues/2
